index.js

```diff
@@ -108,6 +108,10 @@ module.exports = {
     'no-restricted-modules': ['error', { paths: forbiddenModules }],
     'no-restricted-imports': ['error', { paths: forbiddenModules }],
 
+    // AirBnB has this as an error but error is disruptive during development
+    // and some React builders halt a build when an eslint error occurs
+    'no-unused-vars': 'warn',
```

Comment:
thought, what if we base this off of `NODE_ENV`? if `NODE_ENV=development` warn so you know it's there but can continue on with the task at hand, `NODE_ENV=production` stop and fix the issue 'cause it's time for that now?

Reply:
pre-commit checks might want to run eslint with `NODE_ENV=production` in that setup to avoid users getting to a failed PR build or further to publish to find the issue/roadblock, would that be a breaking change?

Reply:
You're gonna love this. Adding a ternary like that in the eslint makes the unit tests for this repo fail because it creates a branch which seemingly can't be tested so the coverage is 50% and I can't commit.

Reply:
I tried this but it didn't work
```
/* istanbul ignore next */
```

Reply:
@stevensacks have you tried using [`jest.isolateModules`](https://jestjs.io/docs/en/jest-object#jestisolatemodulesfn)?